config/locales/views/assignments/en.yml

```diff
@@ -58,15 +58,15 @@ en:
       populate_repo_confirm: Are you sure you want to add the starter files to your repository? If they already exist in the repository, any changes will be overwritten.
       populate_repo_error: Unfortunately, the starter files could not be added to your repository. Please download them as a zip file instead.
       populate_repo_success: Starter files successfully added to your repository.
-      provided: Starter files have been provided for this assignment
+      provided: Starter files have been provided for this assignment.
       rename: Rename Selected File or Directory
       starter_file_rule: Starter File Assignment Rules
       starter_file_rule_types:
         group: Randomly select one starter file group
         sections: Assign starter file groups by section
         shuffle: Randomly select one top level file or directory from each starter file group
         simple: Assign the default starter file group to all students
-      title: Starter Files
+      title: Starter files
```

Comment:
This change is good, but a bit tricky because now the word "files" is no longer capitalized in the download button, which isn't good. You might need to do some case conversion in the front-end to make this work.